src/cpu/rv64/rvv_matmul.hpp

```diff
@@ -59,6 +59,9 @@ struct rvv_matmul_t : public primitive_t {
             VDISPATCH_MATMUL(attr()->scales_.has_default_values(),
                     VERBOSE_UNSUPPORTED_SCALES_CFG);
 
+            VDISPATCH_MATMUL(attr()->dropout_.has_default_values(),
+                    VERBOSE_UNSUPPORTED_ATTR);
```

Comment:
Please follow [this pattern](https://github.com/uxlfoundation/oneDNN/blob/main/src/cpu/matmul/ref_matmul.hpp#L91-L100) where inside the statement supported attributes are specified.